### ISSUE TITLE
(optional) Provide a list of pipelines for each concourse host to process

### ISSUE THREAD

**Issue by edespino** [dgodd/concourse-summary#4]
We have many development pipelines and a smaller set of production pipelines processing our master branches.  It is the production pipelines that we would like concourse-summary to report on.  This request would allow us to optionally provide a list of pipelines for each concourse host to be processed.

**Comment by dgodd** [comment]
@edespino I have created a branch (see above) with what I think right solve both this issue and the multiple concourse hosts issue. I have added 80% of the code to show an example of defining groups in the manifest, so we have a group key, and that has a list of hosts, each of which has a list of pipelines, each of which shows either all, or a specified set of groups (currently it always shows all groups, but filtering the same as pipelines is the intent)

Please have a look and tell me if this feels like what you want and I will do it properly.

Example setup

``` json
{"test":{"buildpacks.ci.cf-app.com":{"binary-builder":["automated-builds","manual-builds"],"brats":[]},"diego.ci.cf-app.com":{"greenhouse":[]}}}
```

can be seen at http://concourse-summary-crystal.cfapps.io/group/test

**Comment by dgodd** [comment]
@edespino I have committed the above code and closed out both this and #6
